src/main/java/uk/gov/hmcts/juror/api/moj/domain/ReportsJurorPayments.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.juror.api.moj.domain;
+
+import groovy.transform.Immutable;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import static uk.gov.hmcts.juror.api.moj.service.letter.court.CourtLetterService.JUROR_NUMBER;
+import static uk.gov.hmcts.juror.api.validation.ValidationConstants.NO_PIPES_REGEX;
+import static uk.gov.hmcts.juror.api.validation.ValidationConstants.POOL_NUMBER;
+
+
+@Entity
+@Table(name = "reports_juror_payments_summary", schema = "juror_mod")
+@NoArgsConstructor
+@Getter
+@AllArgsConstructor
+@Immutable
+public class ReportsJurorPayments extends Address implements Serializable {
+
+    @Column(name = "trial_number")
+    private String trialNumber;
+
+    @Column(name = "loc_code")
+    private String locCode;
+
+    @Column(name = "attendance_date")
+    private LocalDate attendanceDate;
+
+    @Id
```

Comment:
Should this be a composite key between locCode, attendanceDate and jurorNumber?

Reply:
I couldn't figure out how to annotation a composite Id, but also you're not supposed to be constructing one of these as it's an entity for a view rather than a table

Reply:
Fixed